scripts/etc/init.d/rcS

```diff
@@ -1284,6 +1284,17 @@ if [ "$kernel_module" = true ] ; then
   fi
 fi
 
+# install firmware for wireless chipset (RPi 3 and Zero W)
+if [ "$ifname" != "eth0" ]; then
+  echo -n "Installing firmware for Broadcom wireless network cards... "
+  chroot /rootfs /usr/bin/apt-get -y install firmware-brcm80211 &>/dev/null
+  if [ $? -eq 0 ]; then
+    echo "OK"
+  else
+    echo "FAILED !"
+  fi
+fi
+
```

Comment:
Confirmed.
This just installs another package with firmware blobs, which are used by the driver if needed.
The other sections install the program wpa_supplicant, which is needed as well. It handles the actual wireless connections in user mode for both on-board WiFi and dongles.